config/README.md

```diff
@@ -0,0 +1,13 @@
+as work on serverless picks up we will add config values to these files that
+define how Kibana will run in "serverless" modes. To start Kibana locally with
+this configuration, pass `--serverless={mode}` or run `yarn serverless-{mode}`
+
+valid modes are currently: `es`, `oblt`, and `security`
+
+configuration is applied in the following order, later values override
+ 1. kibana.yml
+ 2. serverless.yml
```

Comment:
@clintandrewhall FYI, I tried the `serverless/all.yml` etc idea and unfortunately git has some [weird restriction](https://git-scm.com/docs/gitignore#_pattern_format) that prevents the .gitignores from working correctly.